identity/webapp/src/utility/auth0.ts

```diff
@@ -1,3 +1,6 @@
+// TODO ADDRESS THIS PROPERLY
+// eslint-disable-next-line
+// @ts-nocheck
```

Comment:
Same here; I didn't feel comfortable fixing this as part of this work + the testing needed would require a lot of time so I think it's a different job, but it needs to be addressed or a ticket created for it.